packages/core/src/components/StepList/StepList.scss

```diff
@@ -0,0 +1,291 @@
+@import "@cmsgov/design-system-support/src/settings/index";
+
+/*
+Step List
```

Comment:
One thing currently not implemented are the responsive styles. Here was the original design for that:

<img src="https://user-images.githubusercontent.com/371943/33441104-d9701c7a-d5bf-11e7-9c2b-2bf1465debff.png" width="320" />

The font sizes in the mobile version are:

| Element | Size |
| -------- | ---- |
| Step number | `$small-font-size` | 
| Title | `$h5-font-size` |
| Description | `$small-font-size` | 
| Resume/Completed/Edit | `$small-font-size` |